Assets/GothicVR/Scripts/Creator/SoundCreator.cs

```diff
@@ -114,6 +114,13 @@ private AudioSource CreateAndAddAudioSource(GameObject soundObject, string sound
             else
             {
                 var sfxData = assetCache.TryGetSfxData(soundData.soundName);
+
+                if (sfxData == null)
+                {
+                    Debug.LogError($"No sfx data returned for {soundData.soundName}");
+                    return null;
```

Comment:
Ah! One more thing: Please don't return null! Please set the cache key. Otherwise it will try to load it again and again.